app/controllers/api/v1/orders_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::OrdersController < ApplicationController
+
+  def get_cart
+    render json: cart.to_json
```

Comment:
now that the CartHelper#cart function is called on every page we should change it so that it doesn't create the cart unless we need it to. AKA; if someone hits the homepage but never tries to add anything to their cart it shouldn't save a cart.